examples/sv3d/sgm/modules/optimizers/adamw_zero1.py

```diff
@@ -0,0 +1,169 @@
+import numpy as np
```

Comment:
why add this?

Reply:
Found that this a duplicate of `mindone/trainers/adamw_zero1.py`, removed.